tests/publiccloud/migration.pm

```diff
@@ -66,11 +66,11 @@ sub run {
     # migration finished and instance rebooted
     record_info('INFO', 'Checking the migration succeed');
 
-    my $product_version = $instance->ssh_script_run(cmd => 'sudo cat /etc/os-release');
-    record_info('INFO', $product_version);
+    my $product_version = $instance->run_ssh_command(cmd => 'sudo cat /etc/os-release');
+    record_info('INFO Product Version', $product_version);
```

Comment:
In the openQA frame, long text strings are cut. As a recommendation, I would do one of the following:
`record_info('INFO', "Product Version: $product_version");`
`record_info('Product Version', $product_version);`